Stanford/JumpstartLab/Install/ContextSettings.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Stanford\JumpstartLab\Install;
+
+use \ITasks\AbstractInstallTask;
+
+/**
+ * Class ContextSettings.
+ *
+ * @package Stanford\JumpstartLab\Install
+ */
+class ContextSettings extends AbstractInstallTask {
+
+  /**
+   * Disable some contexts.
+   *
+   * @param array $args
+   *   Installation arguments.
+   */
+  public function execute(&$args = array()) {
```

Comment:
@pookmish,

Are these being disabled because they are not going to be used or is it just to have them disabled to start and then an administrator can enable them later?

If the answer is that they are not to be used you can remove the ones that are found in jsl_layouts module so they don't even get installed. The one found in the people layouts will probably have to stay here then.